tests/parametric/test_headers_asm_standalone.py

```diff
@@ -0,0 +1,166 @@
+        [
+            {
+                "DD_TRACE_PROPAGATION_STYLE": "tracecontext,datadog",
+                "DD_EXPERIMENTAL_APPSEC_STANDALONE_ENABLED": "true",
+                "DD_APPSEC_ENABLED": "true",
+            },
+            {
+                "DD_TRACE_PROPAGATION_STYLE": "datadog,tracecontext",
+                "DD_EXPERIMENTAL_APPSEC_STANDALONE_ENABLED": "true",
+                "DD_APPSEC_ENABLED": "true",
+            },
+        ],
+    )
+    def test_asm_standalone_tracecontext_datadog(self, test_agent, test_library):
+        with test_library:
+            with test_library.dd_extract_headers_and_make_child_span(
+                "identical_trace_info",
+                [
+                    ["traceparent", "00-11111111111111110000000000000001-000000003ade68b1-01"],
+                    ["tracestate", "dd=s:0;p:000000003ade68b1;t.appsec:1,foo=1;s:2"],
+                    ["x-datadog-trace-id", "1"],
+                    ["x-datadog-tags", "_dd.p.tid=1111111111111111,_dd.p.appsec=1"],
+                    ["x-datadog-parent-id", "987654321"],
+                    ["x-datadog-sampling-priority", "2"],
+                ],
+            ) as s1:
+                pass
+
+        traces = test_agent.wait_for_num_traces(2)
+        span = find_span_in_traces(traces, s1.trace_id, s1.span_id)
+        span["name"] = "identical_trace_info"
+        assert span["metrics"].get(SAMPLING_PRIORITY_KEY) == 2
+        assert "_dd.p.appsec" in span["meta"]
+
+    @pytest.mark.parametrize(
+        "library_env",
+        [
+            {
+                "DD_TRACE_PROPAGATION_STYLE": "tracecontext,datadog",
+                "DD_EXPERIMENTAL_APPSEC_STANDALONE_ENABLED": "true",
+                "DD_APPSEC_ENABLED": "true",
+            },
+            {
+                "DD_TRACE_PROPAGATION_STYLE": "datadog,tracecontext",
+                "DD_EXPERIMENTAL_APPSEC_STANDALONE_ENABLED": "true",
+                "DD_APPSEC_ENABLED": "true",
+            },
+        ],
+    )
+    def test_asm_standalone_tracecontext_datadog_without_appsec_tag(self, test_agent, test_library):
+        with test_library:
+            with test_library.dd_extract_headers_and_make_child_span(
+                "identical_trace_info",
+                [
+                    ["traceparent", "00-11111111111111110000000000000001-000000003ade68b1-00"],
+                    ["tracestate", "dd=s:0;p:000000003ade68b1,foo=1"],
+                    ["x-datadog-trace-id", "1"],
+                    ["x-datadog-tags", "_dd.p.tid=1111111111111111"],
+                    ["x-datadog-parent-id", "987654321"],
+                    ["x-datadog-sampling-priority", "0"],
+                ],
+            ) as s1:
+                pass
+
+        trace = test_agent.wait_for_num_traces(2)
+        span = find_span_in_traces(trace, s1.trace_id, s1.span_id)
+        span["name"] = "identical_trace_info"
+        assert span["metrics"].get(SAMPLING_PRIORITY_KEY) == -1
```

Comment:
nodejs tracer is using `AUTO_REJECT (0)` in that case.

Could we change the tests asserts to `assert span["metrics"].get(SAMPLING_PRIORITY_KEY) <= 0`?

Or the nodejs is wrong and that priority value is mandatory?